src/components/GoodCard/GoodCard.jsx

```diff
@@ -0,0 +1,7 @@
+const GoodGard = ({ good }) => (
+  <li data-cy="Good">
+    {good}
+  </li>
+);
+
+export default GoodGard;
```

Comment:
```suggestion
const GoodСard = ({ good }) => (
  <li data-cy="Good">
    {good}
  </li>
);

export default GoodСard;
```

Reply:
It looks the same for me. Could you tell me please what do you mean in this comment?